.github/workflows/pr-container-build-workflow.yaml

```diff
@@ -30,4 +30,4 @@ jobs:
       env:
         CONTAINER_IMAGE_NAME: gh_actions_test
       working-directory: ./apps/${{ matrix.app }}
-      run: docker build -t $CONTAINER_IMAGE_NAME:ci-test .
+      run: docker build -t ${CONTAINER_IMAGE_NAME}:ci-test -f ../../Dockerfile .
```

Comment:
`docker build -t ${CONTAINER_IMAGE_NAME}:ci-test **--build-arg BUILD_ENV=test** -f ../../Dockerfile .` could be better?

Reply:
Since this is a test that the image builds correctly, I'd say keeping this in the default (`ci`, I think) env